### PR TITLE
Social bucket share

### PR DESCRIPTION
**Pull request by alinauroz**
for #378

### CODE REVIEW

**Comment by alinauroz** [comment]
I have used [ngrok](https://ngrok.com/) to test the preview facebook and twitter. Below are the previews

Facebook
Screenshot 2022-10-21 at 12 24 32 AM

Twitter
Screenshot 2022-10-21 at 12 35 05 AM

Linkedin was pulling homepage information ngrok so i copied the *page source* (not using inspect element) and hosted that page on a hosting service. Below is the preview

Screenshot 2022-10-21 at 1 10 06 AM

File: ui/.env [hunk 1]
```diff
@@ -6,4 +6,5 @@ SINGLE_GROUP_MODE=false
 BUCKET_NAME_SINGULAR="bucket"
 BUCKET_NAME_PLURAL="buckets"
 PLATFORM_NAME="Cobudget"
+PLATFORM_LOGO="https://pbs.twimg.com/profile_images/1019572294987976704/j_Ngi0LK_400x400.jpg"
```
Comment (Powersource) [inline ui/.env:9]: better to host this on our server instead

Comment (alinauroz) [reply ui/.env:9]: How to do this?

Comment (aerugo) [reply ui/.env:9]: @alinauroz - if you add it to ui/public it will be available at https://cobudget.com/xxxx.jpg

**Pull request merged**